domkit/CssStyle.hx

```diff
@@ -515,14 +518,16 @@ class CssStyle {
 				if( pr.tag != ntag ) {
 					if( Properties.KEEP_VALUES ) {
 						e.initCurrentValues();
-						e.currentValues.push(p.value);
+						e.currentValues.push(null);

```

Comment:
why null ?

Reply:
In Properties.setAttribute I had to push the value in currentValue and null into currentRuleStyles (because the RuleStyle is not available there, hence the two arrays)

the null here was to consistently have exactly one with null and the other with a filled slot.

Should I change this back to p.value?